modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -46,9 +48,17 @@
 import javax.servlet.jsp.JspException;
 import javax.servlet.jsp.JspWriter;
 
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Modified;
+
 /**
  * @author Marko Cikos
  */
+@Component(
+	configurationPid = "com.liferay.frontend.taglib.clay.internal.configuration.FFManagementToolbarConfiguration",
+	immediate = true, service = BaseContainerTag.class
+)
 public class ManagementToolbarTag extends BaseContainerTag {
```

Comment:
It is a bit weird to convert this Tag to a component just for taking the configuration 🤔 . Does this work? 

I think it may be better to extract this to another class. Somethink like: https://github.com/liferay/liferay-portal/blob/3627c3f7c663a1a3d4f7e564155e33a5390f5219/modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/configuration/FFStyleBookConfigurationUtil.java#L28-L47

Reply:
If it's weird it's because I have no idea what I'm doing with Java in DXP 😞 But it works!
I'll use that as a template for this FF, changing it now. Thank you @victorg1991!

Reply:
Hahaha it is normal, it is not just java it is Java + OSGI + portal stuff :S. You can ping me if you need help with that part :)